src/pages/map/result/Map.tsx

```
@@ -0,0 +1,19 @@
+'use client';
+
+import { Header } from '@/shared/components';
+import router from 'next/router';
+import FullMap from '@/pages/map/result/components/FullMap';
+
+const Map = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)